components/proxy/src/test/scala/com/hotels/styx/proxy/backends/file/FileBackedBackendServicesRegistrySpec.scala

```diff
@@ -190,14 +193,14 @@ class FileBackedBackendServicesRegistrySpec extends FunSpec with Eventually {
         registry.reload()
 
         assertThat(registry.get(), contains(genBackendOne))
-        verifyNoMoreInteractions(listener)
```

Comment:
What is the purpose of this change?

Reply:
To prevent test from failing. `verifyNoMoreInteractions` appears to be picking up some uninteresting interactions from earlier on in the tests, which we did not verify for in this test.